modules/kubernetes/cluster-autoscaler/templates/values.yaml.tpl

```diff
@@ -1,4 +1,4 @@
-cloudProvider: ${provider}
```

Comment:
Cluster autoscaler module should be completely removed as it is not used in Azure. AKS runs this for you.

Reply:
Done, History.